### PR TITLE
material ui

### PR DESCRIPTION
**Pull request by ethanjurman**
IT'S GREAT! BETTER ICONS AND WHATNOT

image

image

image

### CODE REVIEW

**Comment by lukecoy** [comment]
ugh

**Comment by JRJurman** [comment]
ignore failing tests for now until PR that updates the tests is merged in

File: www/js/components/add-point-amenities.js [hunk 1]
```diff
@@ -39,14 +39,15 @@ export class AddPointAmenities extends Component {
     });
 
     let addAmenityButton = (
-      <Button colored
+      <RaisedButton secondary
+              disabled={!(this.state.amenity || this.props.newPoint.amenities.includes(this.state.amenity))}
```
Comment (ethanjurman) [inline www/js/components/add-point-amenities.js]: is includes a function on array?? Double check this logic

File: www/js/components/add-point-hours.js [hunk 2]
```diff
@@ -16,14 +12,19 @@ export class AddPointHours extends Component {
   constructor(props) {
     super(props);
     this.state = {
```
Comment (ethanjurman) [inline www/js/components/add-point-hours.js]: Should this be stateless? @JRJurman

File: www/js/components/add-point-amenities.js (hunk 1 above)
Comment (JRJurman) [inline www/js/components/add-point-amenities.js]: thanks for pointing this out, it's not always, I'll revise this

File: www/js/components/add-point-hours.js (hunk 2 above)
Comment (JRJurman) [inline www/js/components/add-point-hours.js]: I don't think we can make this stateless, unless we want to add an intermittent state for an hours being added (besides, this logic might change with the appropriate new UI).

**Comment by JRJurman** [comment]
:+1: looks good to me!

**Pull request merged**